opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -331,6 +331,12 @@ def __init__(
     def attributes(self) -> types.Attributes:
         return self._attributes
 
+    @property
+    def dropped_attributes(self) -> int:
```

Comment:
Would be nice to add a unit test of this for the two cases

Reply:
Added it in the new commit.